template.yaml

```diff
@@ -201,6 +286,28 @@ Resources:
             Path: /
             Method: post
             RestApiId: !Ref NvaDoiRequestsApi
+  NvaUpdateDoiRequestHandler:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: update-doi-request
+      Handler: no.unit.nva.doi.requests.handlers.UpdateDoiRequestHandler::handleRequest
+      Runtime: java11
+      MemorySize: 1400
+      Environment:
+        Variables:
+          ALLOWED_ORIGIN: '*'
+          TABLE_NAME: !Ref DoiRequestsTableName
+          INDEX_NAME: !Ref DoiRequestsIndexName
+      Policies:
+        - DynamoDBCrudPolicy:
+            TableName: !Ref DoiRequestsTableName
+      Events:
+        GetEvent:
+          Type: Api
+          Properties:
+            Path: /{publicationIdentifier}/
```

Comment:
Trailing slash?

Reply:
Typically not, according to the docs, the path MUST begin with '/', but it doesn't make sense to have the trailing slash here, does it?